.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length = 80
```

Comment:
Are we going to keep this strict ?

Reply:
More info here: https://github.com/psf/black#line-length

Reply:
Yeah, but we talked about raising to 88 and maybe use the B check with tolerance, isn't it?